Gems/EMotionFX/Code/EMotionFX/CommandSystem/Source/AnimGraphNodeGroupCommands.cpp

```diff
@@ -80,78 +110,51 @@ namespace CommandSystem
     }
 
 
-    bool CommandAnimGraphAdjustNodeGroup::Execute(const MCore::CommandLine& parameters, AZStd::string& outResult)
+    bool CommandAnimGraphAdjustNodeGroup::Execute(const MCore::CommandLine& /*parameters*/, AZStd::string& outResult)
```

Comment:
Use [[maybe_unused]] instead?

Reply:
No, I don't want it "maybe" unused. I want it "definitely" unused.

Reply:
I don't know you could do that, good practice?

Reply:
AZ_UNUSED() then?

Reply:
Then you should either use [[maybe_unused]] or delete the parameter.
Don't leave it commented out.
For users of Visual Studio, intellisense doesn't capture commented out parameters when hovering over the function at a call stie, so there is no point in commenting out the parameter.

Reply:
`[[maybe_unused]]` will still allow the parameter to be used, so that's not an option.
Leaving the parameter commented out, but still present, gives a name to the parameter that is not being used, which improves readability.

Reply:
Readability occurs at the function header level.
The commented parameter on the function implementation provides no benefits at all.

What is gained by having a commented out parameter in the function definition?
- If the user wanted to use it, they would have to modify the function definition line anyway to uncomment it.

Reply:
I deleted the parameter from the function signature in the function definition.